Src/VimCore/EditorUtil.fs

```diff
@@ -21,18 +21,60 @@ module internal EditorCoreUtil =
     let IsEndPoint (point: SnapshotPoint) = 
         point.Position = point.Snapshot.Length
 
+    /// TODO: Should this handle surrogate pairs and linebreaks?

```

Comment:
Definitely need the variants which iterate points forward without surrogate pairs context. If we need a suggorate pair move forward that should probably be a different method.

Reply:
I suspected as much, and didn't risk attempting to change it. But can you give me any practical example of a situation where VsVim needs to be able to have a snapshot point pointing to the second character of a surrogate pair? I could not think of any.

Reply:
Removed those comments.